src/toast/atm.py

```diff
@@ -1133,8 +1133,8 @@ def _initialize_kolmogorov(self):
         self._nr = 1000
 
         self._rmin_kolmo = 0.0
-        diag = np.sqrt(self._delta_x ** 2 + self._delta_y ** 2)
-        self._rmax_kolmo = np.sqrt(diag ** 2 + self._delta_z ** 2) * 1.01
+        diag = np.sqrt(self._delta_x**2 + self._delta_y**2)
```

Comment:
Looks like `black` version 22.1 changes the default on space around `**`. Given that `pip` still installs `black` version 21.12. we should expect some noise from this in the commit history. Or should we require 22.1 to run `format_source.sh`?

Reply:
Resolved offline, latest PyPI version is 22.1.  I think our intention is to track the defaults provided by latest `black`, because it relieves us of having to think about it.  However, I agree it does make some commit noise when those defaults change.